### ISSUE TITLE
Only one image is imported from a set

### ISSUE THREAD

**Issue by egermani** [alicevision/meshroom#271]
I'm trying to scan a small statue, and when I drag my photoset into the GUI, only one image makes it in. At the bottom, I've included the log output in the command window that runs alongside the GUI. Before I paste that in, though, my only idea is that something I did when editing my photos confused Meshroom. Unfortunately, I had half my shots in RAW and half in JPG (ran out of space on the memory card), so I converted them all to JPG and cropped them so the figure was just about the only thing in the picture. When I tried importing the source RAWs and source JPGs, they went in okay, but Meshroom didn't like 50% of them. So I thought I'd normalize the lighting and alignment, and that's when this bug occurred. The metadata on the images look the same to me, so I don't think that's the culprit.

Here's an [example-image jpg](https://user-images.githubusercontent.com/13008271/46900788-a59bac00-ce6d-11e8-8a73-9cfe2d627305.jpg) from my edited set, and here's an animation to give you an idea of what they all look like. 
![full-stack](https://user-images.githubusercontent.com/13008271/46900808-fc08ea80-ce6d-11e8-83fa-b4b4c12e1f0d.gif)

Let me know if I can provide any more information, and thanks for this program!

```
Program called with the following parameters:
 * allowIncompleteOutput = 1
 * allowSingleView = 1
 * defaultCameraModel = "" (default)
 * defaultFieldOfView = 45
 * defaultFocalLengthPix = -1 (default)
 * defaultIntrinsic = "" (default)
 * groupCameraModel = 2 (default)
 * imageFolder = "" (default)
 * input = "C:\Users\erikg\AppData\Local\Temp\tmp1k28x4ib/CameraInit/2731003bd822aa3d98acd3185af7e6a65750fcd4//viewpoints.sfm"
 * output = "C:/Users/erikg/AppData/Local/Temp/tmp1k28x4ib/CameraInit/2731003bd822aa3d98acd3185af7e6a65750fcd4/cameraInit.sfm"
 * sensorDatabase = "C:\Users\erikg\Downloads\Meshroom-2018.1.0-win64\Meshroom-2018.1.0\aliceVision\share\aliceVision\cameraSensors.db"
 * verboseLevel = "info"

[22:19:30.094306][info] CameraInit report:
        - # views listed in SfMData: 1
        - # views with an initialized intrinsic listed in SfMData: 1
        - # intrinsics listed in SfMData: 1
```

**Comment by natowi** [comment]
> so I converted them all to JPG and _cropped them_
Try again without cropping.
To get more images included in the reconstruction click on the FeatureExtraction node:
- add AKAZE as DescriberType
- DescriberPreset to High or Ultra

**Comment by egermani** [comment]
Hi natowi, thanks for your reply -- it turned out to be a metadata problem, not a crop problem.

I tried a few different things that I'll mention here in case anyone else finds this issue in the future.

- I tried again without cropping, same bug. 
- I went back to the source raw files and reexported them as `.jpg`, and when I tried to import, Meshroom logged: `Not a JPEG file: starts with 0x89 0x50`
- I exported from Photoshop with "Save for Web (Legacy)", and made sure Metadata was set to "All". That worked! I was also able to get it to work when doing an "Export as...". Meshroom didn't think there was any metadata w/ that second technique, but that was okay, because I wasn't gonna do the Save for Web process 70 times over.

Your tip about the reconstruction was a good one too, thank you very much.
